test/e2e/specs/editor/various/is-typing.spec.js

```diff
@@ -1,46 +1,46 @@
 /**
  * WordPress dependencies
  */
-import {
-	clickBlockAppender,
-	createNewPost,
-	showBlockToolbar,
-} from '@wordpress/e2e-test-utils';
-
-describe( 'isTyping', () => {
-	beforeEach( async () => {
-		await createNewPost();
+/**
+ * WordPress dependencies
+ */
+const { test, expect } = require( '@wordpress/e2e-test-utils-playwright' );
+
+test.describe( 'isTyping', () => {
+	test.beforeEach( async ( { admin } ) => {
+		await admin.createNewPost();
 	} );
 
-	it( 'should hide the toolbar when typing', async () => {
-		const blockToolbarSelector = '.block-editor-block-toolbar';
+	test( 'should hide the toolbar when typing', async ( { page, editor } ) => {
+		const blockToolbar = 'role=toolbar[name="Block tools"i]';
```

Comment:
@pavanpatil1, you can create locators once and then reuse them. The Playwright will make sure to return the up-to-date DOM state for it.

```js
const blockToolbar = page.locator( 'role=toolbar[name="Block tools"i]' );

// Later in tests
await expect( blockToolbar ).toBeHidden();

// After different actions
await expect( blockToolbar ).toBeVisible();
```

Reply:
Hi @Mamaduka, Yes did the same [here](https://github.com/WordPress/gutenberg/blob/db90dfffc3dcebf81bfaca56816f9f5c8ddac0be/test/e2e/specs/editor/various/is-typing.spec.js#L15). I have declared the variable and reusing it.

Reply:
@pavanpatil1, it's not exactly the same. You're storing and reusing the selector, not the locator itself.

```js
const blockToolbar = page.locator( 'role=toolbar[name="Block tools"i]' );

// vs

const blockToolbar =  'role=toolbar[name="Block tools"i]';
const blockToolbarLocator = page.locator( blockToolbar );
```